src/app.css

```diff
@@ -2,13 +2,92 @@
 @tailwind components;
 @tailwind utilities;
 
-@font-face {
-  font-family: 'Inter';
-  font-style: normal;
-  font-weight: 400;
-  src: local('fonts/Inter'), format('woff2');
+.force-white-text {
+	color: white !important;
 }
 
-.force-white-text {
-  color: white !important;
+html {
+	color-scheme: dark;
+	--color-white: #ffffff;
+	--color-black: #000000;
+	--color-gray-50: 249 250 251;
+	--color-gray-100: 243 244 246;
+	--color-gray-200: 229 231 235;
+	--color-gray-300: 209 213 219;
+	--color-gray-400: 156 163 175;
+	--color-gray-500: 107 114 128;
+	--color-gray-600: 75 85 99;
+	--color-gray-700: 55 65 81;
+	--color-gray-800: 31 41 55;
+	--color-gray-900: 17 24 39;
+	--color-gray-950: 3 7 18;
+
+	--color-neutral-50: 250 250 250;
+	--color-neutral-100: 245 245 245;
+	--color-neutral-200: 229 229 229;
+	--color-neutral-300: 212 212 212;
+	--color-neutral-400: 163 163 163;
+	--color-neutral-500: 115 115 115;
+	--color-neutral-600: 82 82 82;
+	--color-neutral-700: 64 64 64;
+	--color-neutral-800: 38 38 38;
+	--color-neutral-900: 23 23 23;
+	--color-neutral-950: 10 10 10;
+
+	--color-slate-50: 248 250 252;
+	--color-slate-100: 241 245 249;
+	--color-slate-200: 226 232 240;
+	--color-slate-300: 203 213 225;
+	--color-slate-400: 148 163 184;
+	--color-slate-500: 100 116 139;
+	--color-slate-600: 71 85 105;
+	--color-slate-700: 51 65 85;
+	--color-slate-800: 30 41 59;
+	--color-slate-900: 15 23 42;
+	--color-slate-950: 2 6 23;
+}
+
+html.light {
+	color-scheme: light;
+	--color-white: #000000;
+	--color-black: #ffffff;
+	--color-gray-950: 249 250 251;
+	--color-gray-900: 243 244 246;
+	--color-gray-800: 229 231 235;
+	--color-gray-700: 209 213 219;
+	--color-gray-600: 156 163 175;
+	--color-gray-500: 107 114 128;
+	--color-gray-400: 75 85 99;
+	--color-gray-300: 55 65 81;
+	--color-gray-200: 31 41 55;
+	--color-gray-100: 17 24 39;
+	--color-gray-50: 3 7 18;
+
+	--color-neutral-950: 250 250 250;
+	--color-neutral-900: 245 245 245;
+	--color-neutral-800: 229 229 229;
+	--color-neutral-700: 212 212 212;
+	--color-neutral-600: 163 163 163;
+	--color-neutral-500: 115 115 115;
+	--color-neutral-400: 82 82 82;
+	--color-neutral-300: 64 64 64;
+	--color-neutral-200: 38 38 38;
+	--color-neutral-100: 23 23 23;
+	--color-neutral-50: 10 10 10;
+
+	--color-slate-950: 248 250 252;
+	--color-slate-900: 241 245 249;
+	--color-slate-800: 226 232 240;
+	--color-slate-700: 203 213 225;
+	--color-slate-600: 148 163 184;
+	--color-slate-500: 100 116 139;
+	--color-slate-400: 71 85 105;
+	--color-slate-300: 51 65 85;
+	--color-slate-200: 30 41 59;
+	--color-slate-100: 15 23 42;
+	--color-slate-50: 2 6 23;
```

Comment:
:question: I am guessing the optimal handling of this would be a refactor where we use custom classes instead of translating colors between modes? E.g. adding `primary` `accent` and other classes ourselves?